core/src/autogluon/core/searcher/bayesopt/autogluon/searcher_factory.py

```diff
@@ -234,6 +292,10 @@ def _common_defaults(is_hyperband: bool) -> (Set[str], dict, dict):
         default_options['gp_resource_kernel'] = 'matern52'
         default_options['resource_acq'] = 'bohb'
         default_options['num_init_random'] = 10
+    if is_constrained:
+        default_options['active_metric'] = DEFAULT_METRIC
```

Comment:
See above. If these are internal, there is no value having the user set them. Correct me if I am wrong.

Reply:
Done. They are internal and fixed.